db/migrate/20160621204706_rename_stil_id_on_user.rb

```diff
@@ -0,0 +1,5 @@
+class RenameStilIdOnUser < ActiveRecord::Migration
+  def change
+    rename_column(:users, :stil_id, :student_id)
```

Comment:
> Should we call it the general student_id or lucat_id?, because it will change ;)

Lucat_id is more descriptive, but if you don't want to change it in all the files I'm ok with student_id...